docs/examples/traefik-ingress.md

```diff
@@ -90,129 +87,124 @@ root@k0s-host ➜ curl http://192.168.0.5
 404 page not found
 ```
 
-## Deploy and access the Traefik Dashboard
+## 3. Deploy and access the Traefik Dashboard
 
-Now that you have an available and addressable load balancer on your cluster, 
-you can quickly deploy the Traefik dashboard and access it from anywhere on your local network 
-(provided that you configured MetalLB with an addressable range).
+With an available and addressable load balancer present on your cluster, now
+you can quickly deploy the Traefik dashboard and access it from anywhere on
+your LAN (assuming that MetalLB is configured with an addressable range).
```

Comment:
If this is a tutorial, I think it should tell the user how to do it and assume the user did what was told, not suggest something and start questioning later if they did it or not.

Reply:
Removed the parenthetical remark, to friendly the sentence up and make the assumption that our reader is doing what they should be doing.